easybuild/easyblocks/o/openfoam.py

```diff
@@ -139,9 +139,13 @@ def configure_step(self):
                 self.cfg.update('prebuildopts', "%(name)s_LIB_DIR=$EBROOT%(name)s/lib" % {'name': dep['name'].upper()})
                 self.cfg.update('prebuildopts', "%(name)s_INCLUDE_DIR=$EBROOT%(name)s/include" % {'name': dep['name'].upper()})
         else:
-            scotch = get_software_root('SCOTCH')
-            if scotch:
-                self.cfg.update('prebuildopts', "SCOTCH_ROOT=$EBROOTSCOTCH")
+            for depend in ['SCOTCH', 'METIS', 'CGAL', 'PARAVIEW']:
+                dependloc = None
+                dependloc = get_software_root(depend)
+                if dependloc:
+                    self.cfg.update('prebuildopts', "{0}_ROOT=$EBROOT{0}".format(depend))
```

Comment:
this looks better than the approach with `%(name)s` used above; are you up for changing the other instances of this pattern throughout this easyblock?